pkg/controller/control.go

```diff
@@ -1220,9 +1220,13 @@ func (c *Controller) handleErrorNoLock(err error) error {
 				}
 			}
 
-			if err := c.handleDiskNoSpaceErrorForReplicas(noSpaceErrMap); err != nil {
-				log.WithError(err).Error("All disks of writable replicas are out of space")
-				return err
+			if noSpaceErr := c.handleDiskNoSpaceErrorForReplicas(noSpaceErrMap); noSpaceErr != nil {
+				if noSpaceErr == types.ErrNoSpaceLeftOnDevice {
+					log.WithError(noSpaceErr).Error("All disks of writable replicas are out of space")
+				} else {
+					log.WithError(noSpaceErr).Error("Last replica is writable and others are on a disk that is out of space")
+				}
```

Comment:
Move these logs into `handleDiskNoSpaceErrorForReplicas`

Reply:
Removed it. Thanks